### ISSUE TITLE
Dojo Basic Reading room broken in SamuraiWTF

### ISSUE THREAD

**Issue by JGillam** [SamuraiWTF/samuraiwtf#10]
It's trying to load from http://samurai instead of http://samurai.wtf

```Warning: fopen(): php_network_getaddresses: getaddrinfo failed: Name or service not known in /var/www/html/text-file-viewer.php on line 28
Warning: fopen(http://dojo-basic/readingroom/backdoor.txt): failed to open stream: php_network_getaddresses: getaddrinfo failed: Name or service no known in /var/www/html/text-file-viewer.php on line 28```

**Comment by mgillam** [comment]
One way to fix this might be to configure nginx to redirect the samurai host to samurai.wtf

**Comment by secureideas** [comment]
Fixed in 4.0.1 update to SamuraiWTF-dojo
